src/oscar/apps/basket/views.py

```diff
@@ -54,10 +54,15 @@ def get_shipping_methods(self, basket):
             basket=self.request.basket, user=self.request.user,
             request=self.request)
 
+    def get_default_shipping_address(self):
+        if self.request.user.is_authenticated():
```

Comment:
Should be using `core.compat.is_user_authenticated()` here - calling the method is deprecated.